src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/Macros/NowMacro.cs

```diff
@@ -40,7 +42,7 @@ public void EvaluateConfig(IEngineEnvironmentSettings environmentSettings, IVari
 
                 if (string.IsNullOrEmpty(p.DataType))
                 {
-                    p.DataType = config.DataType;
```

Comment:
Since we already changed `INowMacroConfig.DataType` to `string`, is this necessary?

Reply:
agreed, will revert.